app/assets/stylesheets/workarea/storefront/affirm/components/_affirm_as_low_as.scss

```diff
@@ -0,0 +1,12 @@
+.affirm-as-low-as {}
+.affirm-as-low-as--pdp {}
+.affirm-as-low-as--cart {}
+.affirm-as-low-as--payment {}
+
+  .affirm-as-low-as__subtext {
+    font-weight: bold;
```

Comment:
Our CSS is done with 4-space tabs, wanna copy in the editorconfig from base? https://github.com/workarea-commerce/workarea/blob/master/.editorconfig